src/installation/README.md

```diff
@@ -0,0 +1,185 @@
+
+```bash
+/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+```
+
+To check if Homebrew is installed, run the following command:
+
+```bash
+brew --version
+```
+
+It should return something like this:
+
+```bash
+Homebrew 3.2.0
+Homebrew/homebrew-core (git revision 3b6; last commit 2021-07-26)
+```
+
+## Installing VSCode
+Visual Studio Code is a free source-code editor made by Microsoft for Windows, Linux and macOS. Features include support for debugging, syntax highlighting, intelligent code completion, snippets, code refactoring, and embedded Git. To install VSCode, run the following command:
+
+```bash
+brew install --cask visual-studio-code
+```
+
+To check if VSCode is installed, run the following command:
+
+```bash
+code --version
+```
+
+## Installing Git
+Git is a free and open-source distributed version control system designed to handle everything from small to very large projects with speed and efficiency. It is the most widely used modern version control system in the world today. To start using Git, you need to run the following command:
+
+```bash
+brew install git
+```
+
+To check if Git is installed, run the following command:
+
+```bash
+git --version
+```
+
+To configure Git, run the following commands:
+
+```bash
+git config --global user.name "YOUR_NAME"
+```
+
+```bash
+git config --global user.email "YOUR_EMAIL"
+```
+
+Then donwload the github cli:
+
+```bash
+brew install gh
+```
+
+and authenticate with your github account:
+
+```bash
+gh auth login
+```
+
+to check if it worked:
+
+```bash
+gh auth status
+```
+
+## Installing Node
+Node.js is an open-source, cross-platform, back-end JavaScript runtime environment that runs on the V8 engine and executes JavaScript code outside a web browser. It allows developers to use JavaScript to write command-line tools and for server-side scripting—running scripts server-side to produce dynamic web page content before the page is sent to the user's web browser. To install Node, run the following command:
+
+```bash
+brew install node
+```
+
+To check if Node is installed, run the following command:
+
+```bash
+node --version
+```
+
+## Installing Nvm
+nvm stands for Node Version Manager. It is a tool that allows you to easily install and manage multiple versions of Node.js on a single machine. This is useful for developers who need to work with different versions of Node.js for different projects. To install nvm, run the following command:
+
+```bash
+brew install nvm
+```
+
+To check if nvm is installed, run the following command:
+
+```bash
+nvm --version
+```
+
+To change versions of node, run the following command:
+
+```bash
+nvm install <version>
+```
+
+for example:
+
+```bash
+nvm install 14.17.6
+```
+
+To see which version are available, run the following command:
+
+```bash
+nvm ls-remote
+```
+
+make sure you team is using the same version of node for consistency and to avoid errors.
+
+## Installing Python
```

Comment:
Let’s add Django as well since a couple PMs said they will be using it for their backend.

Reply:
✅ Done, added a Django initial setup guide

Reply:
Sweet thanks! Perhaps it would be better to merge it with the existing Django guide under the "Guides" section, to avoid overlap, I'll leave that up to your discretion!